src/browser/components/MattermostView.jsx

```diff
@@ -230,6 +239,22 @@ const MattermostView = createReactClass({
     );
   },
 
+  extractFileURL(message) {
+    const matched = message.match(/Not allowed to load local resource:\s*(.+)/);
+    if (matched) {
+      return matched[1];
+    }
+    return '';
+  },
+
+  isNetworkDrive(fileURL) {
+    const u = url.parse(fileURL);
+    if (u.protocol === 'file:' && u.host) {
```

Comment:
Probably we should disallow `localhost`, `127.0.0.1`, `::1`. They are actually local resources.